index.html

```diff
@@ -0,0 +1,252 @@
+                solutions, and engage a broad spectrum of voices.
+              </p>
+            </div>
+            <small class="color-dark-text text-center"
+              >Please contact us per Email for any further questions about
+              AGE21!</small
+            >
+            <a href="mailto:usmansbk@gmail.com"
+              ><small>usmansbk@gmail.com</small></a
+            >
+          </div>
+        </div>
+        <aside class="about-logo bg-white bd-tertiary">
+          <div class="content">
+            <h3 class="header-3 text-center color-secondary">
+              Africa Gender Equality Conference 2021 Logo
+            </h3>
+            <div
+              class="divider small-divider bg-primary bd-primary my-12"
+            ></div>
+            <small class="text-center color-dark-text"
+              >The logo of AGE Conference 2021 was decided through the logo
+              competition from 8 June to 7 July.</small
+            >
+            <div class="box bd-tertiary">
+              <h3 class="logo">AGE<span>Conf</span></h3>
+            </div>
+          </div>
+        </aside>
+        <aside class="past-conf bg-white">
+          <div class="content">
+            <h3 class="header-3 text-center color-secondary">
+              See the past AGE Conferences
+            </h3>
+            <div
+              class="divider small-divider bg-primary bd-primary my-12"
+            ></div>
+            <small class="text-center color-dark-text">
+              Take a look at the last two AGE Conferences which took place in
+              Malawi and in Accra.
+            </small>
+            <ul class="card-list">
+              <li class="card bg-primary color-white">
+                <a href="#conf">
+                  <img
+                    class="card-image"
+                    src="./images/confs/conf.jpg"
+                    alt="2020 Conference"
+                  />
+                  <div class="card-body">
+                    <h6 class="header-2">2020</h6>
+                    <small>AGE Conference 2020 in Lilongwe Malawi</small>
+                  </div>
+                </a>
+              </li>
+              <li class="card bg-primary color-white">
+                <a href="#conf">
+                  <img
+                    class="card-image"
+                    src="./images/confs/conf.jpg"
+                    alt="2019 Conference"
+                  />
+                  <div class="card-body">
+                    <h6 class="header-2">2019</h6>
+                    <small>AGE Conference 2019 in Accra Ghana</small>
+                  </div>
+                </a>
+              </li>
+            </ul>
+          </div>
+        </aside>
+      </section>
+      <footer class="bg-dark color-white footer">
+        <small class="text-center"
+          >2021 Africa Gender Equality. Some Rights Reserved.</small
+        >
+      </footer>
```

Comment:
- [ ] The footer should contain a logo aligned to the left and text aligned to the right. The logo is completely missing. Kindly add the logo and align the text to the right.
- [ ] On mobile size, the footer should have a white background. The dark background should only be applicable to desktop screen sizes.

Reply:
@vichuge here is why my footer is white on mobile and dark on desktop